cli/module_registry.go

```diff
@@ -243,7 +243,7 @@ func UploadModuleAction(c *cli.Context) error {
 
 	tarballPath := moduleUploadPath
 	if !isTarball(tarballPath) {
-		tarballPath, err = createTarballForUpload(moduleUploadPath, c.App.Writer)
+		tarballPath, err = createTarballForUpload(moduleUploadPath, c.App.ErrWriter)
```

Comment:
Justification: Most cli tools that I know of put progress bars / info on stderr instead of stdout.